entities/api/files.py

```diff
@@ -476,6 +482,24 @@ async def create_or_upload_file(
         )
 
 
+@strawberry.mutation(extensions=[DependencyExtension()])
+async def upload_temporary_file(
```

Comment:
Allow the system to create temporary files without creating File or Entity objects. As discussed in a nextgen meeting, this is to support uploading phylotrees from the user that are one-time use